lib/dog_collar/contrib/rails/lograge/delegating_logger.rb

```diff
@@ -9,12 +9,18 @@ module Lograge
         class DelegatingLogger < DogCollar::Logging::Delegator
           %i[debug info warn error fatal].each do |method_name|
             define_method(method_name) do |data|
-              logger.send(method_name, build_message(**data), data)
+              logger.send(method_name, message_string(data), data)
```

Comment:
DogCollar [logging API](https://github.com/getsmarter/dog_collar#logging) takes at most two arguments: a string message and an optional hash containing the log tags.

The delegating logger is supposed to wrap a `DogCollar::Logging::Logger`; in the case `data` is a hash, you're passing the wrapped log method a string and a hash (which is fine). But in the case `data` is a string, you're passing it two strings. 

This means the following code will now break.

```ruby
io = StringIO.new
logger = DogCollar::Contrib::Rails::Lograge::DelegatingLogger(DogCollar::Logging::Logger.new(io))
logger.debug('hello')
```

I would expect in the case that `data` is a string, you pass through just the string and omit the second argument.